views/pages/GraphPage.jsx

```diff
@@ -500,12 +463,10 @@ class Graph extends Component {
       })
       .then((res) => {
         if (res.status === 201) {
-          window.location.replace(
-            routes.subjectView(
-              this.state.subject.project,
-              this.state.subject.sid
-            )
-          )
+          this.setState({
+            preferences: { ...this.state.preferences, config: configurationId },
```

Comment:
why do we set the config within the preferences?

Reply:
The preferences have a config property that's used as the users currently used configuration